modules/ROOT/pages/stats-monitoring.adoc

```diff
@@ -208,6 +208,17 @@ This metric can be used to calculate the latency of `_changes` request.
 
 `_changes` request latency = `request_changes_time` / `request_changes_count`
 
+===== rev_send_latency
+
+In a pull replication, Sync Gateway sends a `/_changes` request to the client.
+The client responds with the list of revisions that it wants to receive.
+
+`rev_send_latency` is measuring the time between the client asking for some revisions via the `/_changes` response, and Sync Gateway sending that revision to the client.
+
+NOTE: Measuring time from the `/_changes` response means that this stat will vary significantly depending on the changes batch size.
+A larger batch size will result in a spike of this stat, even if the processing time per revision is unchanged.
+A more useful stat might be the average processing time per revision.
```

Comment:
@adamcfraser @bbrks do we have a dedicated metric for "average processing time per revision"?

Reply:
Yep! `rev_processing_time` is what you're looking for :)

Reply:
Updated in https://github.com/couchbase/docs-sync-gateway/pull/65/commits/152f5abb8a4d52c47311120b13f1010177e5175c thanks!